templates/layout/sidebar_subscriptions.html.twig

```diff
@@ -1,30 +1,78 @@
-<div class="sidebar-subscriptions section {{ app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_GENERAL_SHOW_SUBSCRIPTIONS_IN_SEPARATE')) is not same as 'true' ? 'inline' : 'section--top' }}">
-    <h3>{{ 'subscriptions'|trans }}</h3>
-    <div class="subscription-list">
-        {% for magazine in magazines %}
-            <div class="subscription {{ openMagazine and openMagazine.name is same as magazine.name ? 'active' : '' }}">
-                <a href="/m/{{ magazine.name }}">
+<aside class="sidebar-subscriptions {{ app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_SUBSCRIPTIONS_IN_SEPARATE_SIDEBAR')) is not same as 'true' ? 'inline' : '' }}">

```

Comment:
small nitpick, could we add a `{% if magazines|length %}` and `{% endif %}` at the end? I _think_ that should exclude this template if the user has no subscriptions. not a big deal but seems like it can be hidden until they sub to something

![image](https://github.com/MbinOrg/mbin/assets/146029455/de1cb2b5-4228-4d4a-b2f0-21c44a340de8)

Reply:
actually that might be confusing with the options in the sidebar still... maybe we can hold off on that for another PR pass, this one has already gotten quite a lot in it

Reply:
I'd put it in a new PR as well. But I'd prefer it if there was just a text inside: "you did not yet subscribe to any magazine".